docs/en/02_Developer_Guides/01_Templates/How_Tos/01_Navigation_Menu.md

```diff
@@ -31,4 +31,4 @@ top level menu with a nested second level using the `Menu` loop and a `Children`
 ## Related
 
 * [Template Syntax](../syntax)
-* [Common Variables](../common_variables)
\ No newline at end of file
+* [Common Variables](../common_variables)
```

Comment:
I believe that's an artifact from editing via the online interface. Github is either adding a new line at the end of the file, or removing it?